src/silo/database.cpp

```diff
@@ -550,6 +552,14 @@ void silo::Database::preprocessing(const PreprocessingConfig& config) {
       config.sequence_file.extension()
    );
 
+   SPDLOG_INFO("preprocessing - validate database config");
```

Comment:
I wonder whether we should actually do that earlier. There is no need to do the whole partitioning when we notice later on that the metadata file was actually invalid.